src/test/scala/mesosphere/marathon/integration/LeaderIntegrationTest.scala

```diff
@@ -27,6 +40,6 @@ class LeaderIntegrationTest extends IntegrationFunSuite with MarathonClusterInte
     (result.entityJson \ "message").as[String] should be ("Leadership abdicted")
 
     And("the leader must have changed")
-    marathon.leader().value shouldNot be(leader)
+    eventually { marathon.leader.value shouldNot be(leader) }
```

Comment:
I like Eventually, but we already have WaitTestSupport. We should strive for consistency.